app/Models/Role.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseModel, column } from '@ioc:Adonis/Lucid/Orm'
+
+export default class Role extends BaseModel {
+  @column({ isPrimary: true })
+  public name: string
```

Comment:
Ich würde definitiv noch die 
`id`, `createdAt` und `updatedAt` hinzufügen?

und bei name kannst du das einschränken mit
`public name: 'admin' | 'moderator' | 'user | 'guest | string`

Reply:
Die Rollen sind doch fix oder? also es werden keine angelegt oder geändert, braucht man dann die timestamps?
Hatten wir uns für die keys nicht auf string geeinigt, wäre die id dan überflüssig.

Reply:
ich habe jetzt als type:
`public name: Roles`
eine role die nicht in den enums drin ist haben wie nicht oder?

Reply:
Wir hatten aber esagt, wir verknüpfen die id https://github.com/GoodNightCouchSide/gncs-api/wiki/Documentation#user-1

Reply:
die Kommentare hatte ich vor unserem Gespräch geschrieben, jetzt ist uuid der key